sample-kotlin/build.gradle

```diff
@@ -27,5 +27,6 @@ dependencies {
     implementation (project(':library')) {
         exclude module: "support-v13"
     }
+    provided project(':processor')
```

Comment:
Oh I thought this causes compile error with the latest android gradle plugin 👀

Reply:
You are right. I need to remove this one.